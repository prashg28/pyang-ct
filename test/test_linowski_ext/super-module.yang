module super-module {
    namespace "uri:sm";
    prefix "sm";
    include submodule;

    import complex-types {
    	prefix ct;
    }

    grouping test {
    }

    ct:complex-type test {
    }

    ct:instance newNode3 {
        ct:instance-type "sm:newType3";
    }

    ct:instance newNode2 {
        ct:instance-type "sm:newType2";
    }
}
