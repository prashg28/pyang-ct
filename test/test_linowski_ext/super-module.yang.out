super-module.yang:10: error: GROUPING_ALREADY_DEFINED
super-module.yang:13: error: COMPLEX_TYPE_ALREADY_DEFINED
super-module.yang:17: error: COMPLEX_TYPE_NOT_FOUND
submodule.yang:7: error: MISSING_INCLUDE
submodule.yang:14: error: COMPLEX_TYPE_ALREADY_DEFINED
submodule.yang:14: warning: UNUSED_COMPLEX_TYPE
submodule.yang:24: warning: UNUSED_COMPLEX_TYPE
subsubmodule.yang:10: error: COMPLEX_TYPE_ALREADY_DEFINED
subsubmodule.yang:14: warning: UNUSED_COMPLEX_TYPE
